### PR TITLE
Transaction Fee Refactor

### PR DESCRIPTION
**Pull request by mikedennis**
- Default mintxfee and fallbackfee from network
- Make minrelayfee configurable and default to network value
- Remove mempool validator static field for minrelayfee

### CODE REVIEW

File: Stratis.Bitcoin/Utilities/NetworkExtensions.cs [hunk 1]
```diff
@@ -10,18 +10,6 @@ public static class NetworkExtensions
         /// <summary>Fake height value used in Coins to signify they are only in the memory pool (since 0.8).</summary>
         public const int MempoolHeight = 0x7FFFFFFF;
 
-        /// <summary>Default minimum transaction fee for bitcoin network.</summary>
```
Comment (bokobza) [inline Stratis.Bitcoin/Utilities/NetworkExtensions.cs]: so where are all these actually set?

Comment (mikedennis) [reply Stratis.Bitcoin/Utilities/NetworkExtensions.cs]: The defaults are now in NStratis in Network when the network is created. This avoids the ugly if (bitcoin) logic.

Comment (bokobza) [reply Stratis.Bitcoin/Utilities/NetworkExtensions.cs]: ah is it in your latest change to NStratis?

Comment (mikedennis) [reply Stratis.Bitcoin/Utilities/NetworkExtensions.cs]: Yes it is. It's already merged into fullnode. https://github.com/stratisproject/NStratis/commit/d897a44114b564e982fb03b3408457725cdb055d

Comment (bokobza) [reply Stratis.Bitcoin/Utilities/NetworkExtensions.cs]: ah ok cool, I'm happy with this.
Can you please just reply to my other comments and I'll push it in.

Comment (mikedennis) [reply Stratis.Bitcoin/Utilities/NetworkExtensions.cs]: There you go!

**Pull request merged**